docs/developers/documentation/index.md

```diff
@@ -340,3 +340,69 @@ You can also reach out to us on [zulip](https://napari.zulipchat.com/#narrow/str
 
 Not sure where to place your document or update `_toc.yml`? Make a best guess and open the pull request - the napari team will
 help you edit your document and find the right spot!
+
+## Building the documentation on Windows
+
+The documentation build requires some linux specific commands, so some extra steps are required to build the documentation on Windows. There are multiple tools for this, but [Git Bash](https://gitforwindows.org/) or [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) is recommended.
+
+### Git Bash
+
+First, you will need to install make on Windows:
```

Comment:
because make is such a common verb, I think using backticks to make clear it's a command/tool makes it clear that it's not some grammar mistake to someone who may not be familiar with it.